src/bucket.ts

```diff
@@ -548,6 +548,28 @@ class Bucket extends ServiceObject {
    *   .on('data', function(file) {
    *     this.end();
    *   });
+   *
+   * //-
+   * // If you're filtering files with a delimiter, you should use
+   * // {@link Bucket#getFiles} and set `autoPaginate: false` in order to
+   * // preserve the `apiResponse` argument.
+   * //-
+   * const prefixes = [];
+   *
+   * function callback(err, files, nextQuery, apiResponse) {
+   *   prefixes = prefixes.concat(apiResponse.prefixes);
+   *
+   *   if (nextQuery) {
+   *     bucket.getFiles(nextQuery, callback);
+   *   } else {
+   *     // prefixes = The finished array of prefixes.
+   *   }
+   * }
+   *
+   * bucket.getFiles({
+   *   autoPaginate: false,
+   *   delimiter: '/'
+   * }, callback);
```

Comment:
Could we incorporate an example as well?
```
### Example
For example, I have a bucket with the following object names:
Remember each of the following names are separate names with the delimiter in the name.
Note: pre-chunking on the part of the name you might want to search on. 
 
- a
- a/b/c/d
- b/d/e
 
#### First Request
I call list_blobs with `delimiter="/" and prefix=None`, and GCS will respond with the following list of blobs:
 
- a
 
In `blobs.prefixes`, you'll now find additional information on available prefixes "or" subdirectories exist at this level:
 
- a/
- b/
 
#### Second Request
 
Now, I have a list of prefixes that I can continue opening similar to a filesystem. 
Next, I'll go into `prefix="a/" and delimiter="/"`.
List of Blobs:
- Empty (there are no objects in this directory)
 
List of Prefixes (sub-directories):
- a/b/
 
At this point I can continue making requests to simulate a file system using `prefix="a/b/" and delimiter="/"`. 
```

Reply:
Sorry, forgot to tag you here. On it!

Reply:
@frankyn I added this in this commit: https://github.com/googleapis/nodejs-storage/pull/892/commits/f415824d73509f9da17f179711d2a3e8b4da4dd9 -- PTAL!

Just a note about why there are two documentation areas. The documentation for the streaming method, `getFilesStream()`, makes a mention of how to use the non-streaming `getFiles()` with prefixes/delimiters. Then, `getFiles()` has the more elaborate documentation.